src/components/ErrorMessage/index.jsx

```diff
@@ -0,0 +1,14 @@
+import "./index.css";
+
+const ErrorMessage = (props) => {
```

Comment:
Once you conditionally render the error, since it's just one h2 tag, we might not need this component.